src/features/content-answers/body/__tests__/WelcomeMessage.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { shallow } from 'enzyme';
```

Comment:
should be using react-testing-library for all tests?

Reply:
I am not sure if this is required to only use react-testing-library. @tjuanitas Do you recall if there is any specific requirement? If so, I could check whether the correct text is rendering in the document instead.

Reply:
if we're already writing tests with react-testing-library, then we might as well stay consistent with that throughout this component